pages/_app.tsx

```diff
@@ -84,13 +84,14 @@ const App = ({ Component, pageProps }: AppProps) => { // eslint-disable-line
                         <GraphProvider>
                             <FactoryStore>
                                 <TransactionStore>
-                                    {process.env.NEXT_PUBLIC_DEPLOYMENT === 'DEVELOPMENT' ? (
-                                        <Component {...pageProps} />
-                                    ) : (
-                                        <WhitelistBlock>
-                                            <Component {...pageProps} />
-                                        </WhitelistBlock>
-                                    )}
+                                    <Component {...pageProps} />
+                                    {/*{process.env.NEXT_PUBLIC_DEPLOYMENT === 'DEVELOPMENT' ? (*/}
+                                    {/*    <Component {...pageProps} />*/}
+                                    {/*) : (*/}
+                                    {/*    <WhitelistBlock>*/}
+                                    {/*        <Component {...pageProps} />*/}
+                                    {/*    </WhitelistBlock>*/}
+                                    {/*)}*/}
```

Comment:
Would prefer to keep this in a separate PR

Reply:
If you prefer if it were added back I could do, It was mainly so that it was in line with the user flow on Miro

Reply:
Not sure if the whitelist is serving any purpose at the moment

Reply:
its still just blocking users. There are still a few users interacting with the app but we dont want everyone too. But because production is master now changes to develop wont effect master